apis/externalsecrets/v1beta1/secretsstore_bitwarden_types.go

```diff
@@ -0,0 +1,46 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	esmeta "github.com/external-secrets/external-secrets/apis/meta/v1"
+)
+
+// BitwardenSecretsManagerProvider configures a store to sync secrets with a Bitwarden Secrets Manager instance.
+type BitwardenSecretsManagerProvider struct {
+	APIURL             string `json:"apiURL,omitempty"`
+	IdentityURL        string `json:"identityURL,omitempty"`
+	BitwardenServerSDK string `json:"bitwardenServerSDK,omitempty"`
```

Comment:
We need the URL for the bitwarden server, because we can't easily and reliably detect it. The service name is based on chart deployment. I was thinking of changing it to something permanent, however, that would subtract from the versatility of deploying this server where ever the custom wants it to be.